smartmin/users/views.py

```diff
@@ -274,6 +275,22 @@ class Forget(SmartFormView):
         success_url = "@users.user_login"
         fields = ('email', )
 
+        def build_email_context(self):
+            context = Context()
+
+            processors = []
+            collect = []
+            collect.extend(('smartmin.users.context_processors.links_components',))
```

Comment:
I would say that the methods should probably have a signature like this as well:

```def build_email_context(request=None, user=None)```

Where they need to support either both being None. (thinking of the case of say using this method from a celery task)